integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisPolicyServiceIntegrationTest.java

```diff
@@ -100,6 +96,11 @@ public class PolarisPolicyServiceIntegrationTest {
   private static final String CATALOG_ROLE_1 = "catalogrole1";
   private static final String CATALOG_ROLE_2 = "catalogrole2";
   private static final String EXAMPLE_TABLE_MAINTENANCE_POLICY_CONTENT = "{\"enable\":true}";
+  private static final String INVALID_NAMESPACE = "INVALID_NAMESPACE";
+  private static final String INVALID_TABLE = "INVALID_TABLE";
+  private static final String INVALID_POLICY = "INVALID_POLICY";
+  private static final String INVALID_NAMESPACE_MSG =
+      String.format("Namespace does not exist: %s", INVALID_NAMESPACE);
```

Comment:
It might be good to shift these message constants below rather than being in the middle of the entity names